coqbot.toml

```diff
@@ -0,0 +1,18 @@
+[bot]
```

Comment:
What about renaming this file to something else than `coqbot.toml` or moving it to a subfolder so that if we ever end up using the bot on this repository, this is not mistaken as the repository-specific configuration file?